webapp/portlet/src/main/webapp/space-menu/extension.js

```diff
@@ -2,4 +2,10 @@ function getExtensionsByType(type) {
   return extensionRegistry.loadExtensions('space-menu', type);
 }
 
-export const spaceMenuActionComponents = getExtensionsByType('action-component');
\ No newline at end of file
+export const spaceMenuActionComponents = getExtensionsByType('action-component');
+
+function getExtensionsByTypeCallButton(type) {
```

Comment:
```suggestion
function getSpaceTitleExtensionsByType(type) {
```
In fact, here we are in social, so, no need to refer to something which is in an addon (Call)